samples/bluetooth/bthome_sensor_template/README.rst

```diff
@@ -1,9 +1,13 @@
-.. _bluetooth-bthome-sensor-template-sample:
+.. zephyr:code-sample:: bluetooth_bt_home_sensor_template
```

Comment:
```suggestion
.. zephyr:code-sample:: bluetooth_bthome_sensor_template
```

`bthome` is the name and not `bt_home` :)

Reply:
It's spelled in one word afaict and the folder is also one word so I'm confused? :)

Reply:
Oh, right. That's exactly what you're saying 😅